facia-tool/app/model/frontsapi.scala

```diff
@@ -127,24 +127,19 @@ trait UpdateActions extends Logging with ExecutionContexts {
     FaciaApiIO.putCollectionJson(id, collectionJson)
 
   //Archiving
-  //Archiving
   def archivePublishBlock(collectionId: String, collectionJson: CollectionJson, identity: User): CollectionJson = {
-    FaciaToolArchive.archive(ArchiveRequest(collectionId, identity.email, Json.toJson(collectionJson), Json.obj("action" -> "publish")))
```

Comment:
You're turning off auditing?
If you ever have too look into requests, you will wish you had this.